src/components/svg/Plate96SVG.vue

```diff
@@ -1,8 +1,8 @@
 <template> 
   <div class="plate">
     <svg
-      width="50%"
-      height="50%"
+      width="30%"
```

Comment:
Will this affect the Pacbio plate as well?

Reply:
Yep good point, will add now